packages/fx-core/src/question/create.ts

```diff
@@ -1403,7 +1406,9 @@ export function getLanguageOptions(inputs: Inputs): OptionItem[] {
     if (
       projectType === ProjectTypeOptions.outlookAddin().id ||
       (projectType === ProjectTypeOptions.officeXMLAddin().id &&
-        host === OfficeAddinHostOptions.outlook().id)
+        host === OfficeAddinHostOptions.outlook().id) ||
+      capabilities === CapabilityOptions.officeAddinImport().id ||
+      capabilities === CapabilityOptions.outlookAddinImport().id
```

Comment:
when capability = `outlook-addin-import` or `office-addin-import`, programming-language will not show